chrome-launcher/chrome-launcher.ts

```diff
@@ -75,21 +104,14 @@ export class ChromeLauncher {
     return flags;
   }
 
-  prepare() {
-    switch (process.platform) {
-      case 'darwin':
-      case 'linux':
-        this.TMP_PROFILE_DIR = unixTmpDir();
-        break;
-
-      case 'win32':
-        this.TMP_PROFILE_DIR = win32TmpDir();
-        break;
-
-      default:
-        throw new Error('Platform ' + process.platform + ' is not supported');
+  private prepare() {
+    const platform = process.platform as SupportedPlatforms;
+    if (_SUPPORTED_PLATFORMS.indexOf(platform) === -1) {
```

Comment:
was going to use a set here which seems the best but support in node is lacking :(